### ISSUE TITLE
Environmental variables are disregarded in exporting Foreman script

### ISSUE THREAD

**Issue by isundaylee** [tomafro/recap#115]
In the task foreman:export, no '--env' option is passed to the foreman command. Therefore no environmental variables are present in the upstart scripts exported. For me this results in the program starting in RAILS_ENV=development. Is this an intentional design or a potential BUG?

**Comment by floehopper** [comment]
I recently ran into this problem too. I've come up with a temporary workaround:

```
set :foreman_template_option, "--env #{application_home}/.env"
```

As @isundaylee says, it would be good to know whether it's _intentional_ that the `foreman:export` task does not load the `.env` file.

If it is intentional, how are the environment variables in `.env` supposed to be loaded by the upstart scripts? I'd be happy to provide a PR once I know what the intention is.

**Comment by lazyatom** [comment]
I don't think this is intentional; at some point, `foreman` itself changed and stopped inheriting environment variables.

I think as well as this fix, you may also need to re-export your foreman-generated startup scripts after _changing_ any environment variables, otherwise the processes won't have those new variables set either. This is certainly true for upstart.

So, my fix is:

``` ruby
set :foreman_template_option, "--env #{application_home}/.env"
after "env:set", "foreman:export"
```
